lib/config_file.sh

```diff
@@ -89,8 +89,11 @@ generate_default_config() {
 	# Overlay directory to look for segments. There you can put your own segments outside the repo. Fallback will still be the "segments" directory in the repo.
 	export TMUX_POWERLINE_DIR_USER_SEGMENTS="\${XDG_CONFIG_HOME:-\$HOME/.config}/tmux-powerline/segments"
 
-	# The initial visibility of the status bar. Can be {"on, off"}.
+	# The initial visibility of the status bar. Can be {"on", "off", "2"}. 2 will create two status lines.
 	export TMUX_POWERLINE_STATUS_VISIBILITY="${TMUX_POWERLINE_STATUS_VISIBILITY_DEFAULT}"
+	# In case of visibility = 2, where to display window status and where left/right status bars. 0 upper, 1 lower.
+	export TMUX_POWERLINE_WINDOW_STATUS_LINE=0
```

Comment:
Are these two options mutually exclusive! If one is 0, must the other be 1?

In that case it’s bloated to expose 2 config names for this, then 1 is enough and derive the other one in the code from the first:).

Reply:
You're right, the way it is implemented right now, they are more or less mutually exclusive.
So we have basically two options now:
1. Make it work, so that we can set both on 0, or both on 1. This would be pretty easy, but right now i could not think of a use case for having a dual-line status bar managed by tmux-powerline, but only using one the lines. 
3. Remove the `$TMUX_POWERLINE_OTHER_STATUS_LINE`

Reply:
I've decided for option 2, as there is no real scenario here right now and if it comes up, we'll implement it properly.

Reply:
Since this is basically a Boolean configuration, I think it would be easiest for the user to have. Single confvar 

```bash
TMUX_POWERLINE_WINDOW_STATUS_SEPARATE=true/false
```

and letting it be `false` by default to keep things simple and opt-in.